0011-herbarium-system.md

```diff
@@ -0,0 +1,78 @@
+# Design of the herbarium system
+
+* Status: proposed 
+<!-- {proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)} --> <!-- optional -->
+* Deciders: @ZavenArra @CammWeb
+<!-- Find deciders here: https://github.com/orgs/Greenstand/people  -->
+* Date: 2022-06-07
+
+Technical Story: 
+* https://github.com/Greenstand/treetracker-herbarium-api/issues/1 
+* https://www.figma.com/file/RrG8fahXei5SDY6AYjYYnn/Digital-Herbarium---Species%2FCommon-Names?node-id=0%3A1
+
+## Context and Problem Statement
+
+Tree planting and tracking organizations use various, localy relevant common names to identify the species of tree they are tracking.  This leads to inconsistencies when identifying which species was actually tracked.  Treetracker requires an herbarium service to connect these common names to the globally recognized scientific name.  The herbarium service can also also server as repository of identifying information and socially or ecologically relevant parameters for tree species tracked by organizations.
+
+
+## Decision Drivers <!-- optional -->
+
+* {driver 1, e.g., a force, facing concern, …}
+* {driver 2, e.g., a force, facing concern, …}
+* … <!-- numbers of drivers can vary -->
+
+## Considered Options
+
+1. Implement an herbarium microservice to provide organization specific common names, while aggregating globally relevant species information.
+2. Alternative option?
```

Comment:
The hidden complexity of is that if we allow each organization to see a short list of local names for their organization, the mapping between each local name and a scientific name and/or taxon code needs to be examine carefully. Who will do this? This multiplies the work of keeping species information accurate many times. For example: an org (orgX) says "we plant ironwood", and either we at GS or the org people say "we think ironwood is taxon Z", and we add "ironwood"@orgX to the names for taxon Z without checking, and then we server "ironwood" as a name for orgX in the admin panel. Unless `"ironwood"@orgX = taxon Z` has been checked, we may just be adding garbage to the DB. To check this, someone needs to look at several seedlings of "ironwood" owned by orgX.  To do this checking for every species x org combination is a huge amount of work.  The current systems places the onus on the org, to make their own matching of their local name to a taxon code.
```suggestion
2. Offer a set of standardized taxa codes and names, filtered by the planting list provided by an organization.
```

Reply:
I think 0012 is the alternative option here, right?

Reply:
Yes

Reply:
Great.